components/media-carousel/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "media-carousel",
-  "version": "0.0.13",
+  "version": "0.0.15",
```

Comment:
I increment this from 0.0.13 to 0.0.15 because I incremented it to 0.0.14 in an earlier PR #239